### ISSUE TITLE
InputEventMouseMotion events are only received in `_input(event)` if game window loses and receives focus before

### ISSUE THREAD

**Issue by BeayemX** [godotengine/godot#53894]
### Godot version

v4.0.dev.custom_build [c24027548]

### System information

Linux Mint - Vulkan API 1.2.131 - Using Vulkan Device #0: NVIDIA - NVIDIA GeForce GTX 1080

### Issue description

`InputEventMouseMotion` events are not received when starting the game.
Other events like keyboard or mouse clicks do work.

The game window must lose and receive the focus again first, before `InputEventMouseMotion` events are received.

### Steps to reproduce

1. Put this script on a Node2D:
```
extends Node2D

func _input(event):
	print(event)
```
2. Start the scene
3. Move the mouse around and see that nothing is printed
4. Click into the window, see these messages printed:
```
InputEventMouseButton: button_index=1, mods=none, pressed=true, position=((424, 313)), button_mask=1, double_click=false
InputEventMouseButton: button_index=1, mods=none, pressed=false, position=((424, 313)), button_mask=0, double_click=false
```
5. Move mouse again, see that even though window is focused, no prints are created
6. Select a different window
7. Select the game windows again
8. Move mouse and see that prints now appear as expected

(Instead of step 6. and 7. it also works to click on the title bar of the game window)

### Related issues
Maybe related to #49978 ?

**Comment by Calinou** [comment]
@BeayemX Can you reproduce this on 3.3.4 or 3.4 beta6?

**Comment by Calinou** [comment]
I can confirm this on commit c24027548. This prevents editor freelook and panning from working until you unfocus the editor window and focus it again.

**Comment by superlou** [comment]
I confirmed with a [build from 11/18/21](https://downloads.tuxfamily.org/godotengine/testing/4.0/4.0-dev.20211117/Godot_v4.0-dev.20211117_linux.64.zip). It's definitely about the game window losing focus and then gaining focus again.

**Comment by Jimmio92** [comment]
I didn't see this bug when I originally searched before reporting this same problem; my apologies. Some others tested and it seems to work as is on KDE, while not working without moving the window/refocusing on XFCE. It definitely sounds like that focus problem is the real cause. I have since closed #57294 .

**Comment by Unknown-User2** [comment]
This happens on dwm too
